srcpkgs/deadbeef/template

```diff
@@ -1,11 +1,11 @@
 # Template file for 'deadbeef'
 pkgname=deadbeef
-version=1.8.3
+version=1.8.7
 revision=1
 build_style=gnu-configure
-configure_args="--disable-oss $(vopt_if gtk3 --disable-gtk2 --disable-gtk3)"
+configure_args="--disable-oss --disable-lfm --disable-notify $(vopt_if gtk3 --disable-gtk2 --disable-gtk3)"
```

Comment:
notify seems to depend only on dbus, why disable?

Reply:
Their notify plugin depends on libdispatch (which we don't ship).